lib/activerecord-import/import.rb

```diff
@@ -101,7 +101,12 @@ def bulk_import(*args, &block)
 
     model_klass = reflection.klass
     symbolized_foreign_key = reflection.foreign_key.to_sym
-    symbolized_column_names = model_klass.column_names.map(&:to_sym)
+
+    symbolized_column_names = if model_klass.connection.respond_to?(:supports_virtual_columns?) && model_klass.connection.supports_virtual_columns?
+      model_class.columns.reject(:virtual?).map { |c| c.name.to_sym }
```

Comment:
@jkowens I think you mean `model_class.columns.reject(&:virtual?).map { |c| c.name.to_sym }` here

Reply:
I sure do, thanks!